genotype_api/services/snp_service/snp_service.py

```diff
@@ -0,0 +1,37 @@
+"""Module to hold the snp service."""
+
+from fastapi import UploadFile
+
+from genotype_api.database.crud.create import create_snps
+from genotype_api.database.crud.delete import delete_snps
+from genotype_api.database.crud.read import get_snps_by_limit_and_skip, get_snps
+from genotype_api.database.models import SNP
+from genotype_api.dto.snp import SNPResponse
+from genotype_api.exceptions import SNPExistsError
+from genotype_api.services.snp_reader_service.snp_reader import SNPReaderService
+
+
+class SNPService:
+
+    def __init__(self, session):
+        self.session = session
+
+    @staticmethod
+    def _get_snp_response(snp: SNP):
+        return SNPResponse(ref=snp.ref, chrom=snp.chrom, pos=snp.pos, id=snp.id)
+
+    def get_snps(self, skip: int, limit: int) -> list[SNPResponse]:
```

Comment:
I would consider this change to be consistent with the return type
```suggestion
    def get_snp_responses(self, skip: int, limit: int) -> list[SNPResponse]:
```

Reply:
The idea of the service layer between the CRUD and the API layers is to translate database objects into response objects. This is to detach the CRUD from the API. This layer always should return some kind of response that can be used in the API. 

I was thinking of further refactoring the API services to make this more clear in a subsequent PR. I will address this comment there then if that is ok with you.